zelda-totk/zelda-totk.css

```diff
@@ -929,3 +929,133 @@ label.clickable:has(> input[type=radio]:checked){
 .tooltip.position-horizontal.align-top .arrow{top:3px}
 .tooltip.position-horizontal.align-center .arrow{top:50%;margin-top:-5px}
 .tooltip.position-horizontal.align-bottom .arrow{bottom:3px}
+
+.disable-pointer{
+  pointer-events: none;
+}
+.editItem{
+  display: inline-block;
+  position: relative;
+  z-index: 90;
+}
+.editItem,
+.editItem .search-input,
+.editItem .search-filter,
+.editItem .search-filter *{
+  pointer-events: auto;
+}
+.editItem .search-filter{
+  padding: 8px 0 8px 8px;
+  width: 225px;
+  max-height: 412px;
+  min-height: 62px;
+  position: absolute;
+  top: 46px;
+  left: 0px;
+  background-color: #121e1a;
+  border-radius: 4px;
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.5);
+  overflow: auto;
+  box-sizing: border-box;
+}
+
+.editItem .search-filter:empty::before{
+  content: "no data";
+  display: block;
+  width: 100%;
+  height: 46px;
+  line-height: 46px;
+  text-align: center;
+  color: rgba(255, 255, 255, 0.5);
+  font-size: 16px;
+  font-style: italic;
+}
+
+.editItem .search-filter::-webkit-scrollbar{
+  width:8px;
+  height:8px;
+}
+.editItem .search-filter::-webkit-scrollbar-track{
+  background: #121e1a;
+  border-radius:0 4px 4px 0;
+}
+.editItem .search-filter::-webkit-scrollbar-thumb{
+  background: #21362f;
+  border-radius:4px;
+}
+
+.editItem .search-filter .option{
+  width: 100%;
+  height: 46px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0px 4px;
+  transition: 300ms;
+  border-radius: 4px;
+  margin-bottom: 4px;
+}
+.editItem .search-filter .option:last-child{
+  margin-bottom: 0;
+}
+/* .editItem .search-filter .option:hover{
+  background-color: rgba(255, 255, 255, .15);
+} */
+.editItem .search-filter .option.active{
+  background-color: rgba(255, 255, 255, .15);
+}
+.editItem .search-filter .option .item-icon{
+  width: 40px;
+  height: 40px;
+  margin-right: 10px;
+  pointer-events: none;
+}
+.editItem .search-filter .option .item-name{
+  font-size: 15px;
+  color: #ffffff;
+  overflow : hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  pointer-events: none;
+}
+.editItem .search-filter .option .item-name:hover{
+  text-decoration: none;
+}
+.editItem .search-input{
+  width: 225px;
+  height: 36px;
+  background-color: #111111;
+  border: solid 1px #444444;
+  border-radius: 4px;
+  font-size: 15px;
+  outline: none;
+  color: #ffffff;
+  text-indent: 8px;
+  box-sizing: border-box;
+  transition: 300ms;
+}
+.editItem .search-input::placeholder{
+  font-size: 15px;
+  color: rgba(255, 255, 255, 0.8);
+}
+.editItem .search-input:hover{
+  border: solid 1px #666666;
+}
+.edit-item-mask{
```

Comment:
Gonna try to get rid of the `edit-item-mask` element, I'm pretty use it's not necessary if we bind the blur event correctly to the search filter input.

Reply:
The reason for using this is that if the option list is clicked by the mouse, it triggers the blur event of the input first, removing the option list before the option is clicked, leading to a failed selection. Perhaps a custom event could be used to achieve this function.